src/main/java/macrobase/analysis/classify/OutlierClassifierImpl.java

```diff
@@ -0,0 +1,27 @@
+package macrobase.analysis.classify;
+
+import macrobase.analysis.result.OutlierClassificationResult;
+import macrobase.conf.MacroBaseConf;
+import macrobase.datamodel.Datum;
+
+import java.util.Iterator;
+
+/**
+ * Consumes a tuple at-a-time from a FeatureTransform and for each tuple outputs
+ * a OutlierClassificationResult that should indicate if the tuple observed was an
+ * outlier or not.
+ */
+public abstract class OutlierClassifierImpl implements OutlierClassifier {
```

Comment:
To share the constructor (the DumperClassifier doesn't use that constructor). Do you think we should just have the constructor explicitly in each classifier since it's very short?